locales/en.json

```diff
@@ -104,6 +104,12 @@
       "question": "Do you really want to delete a Following",
       "btnYes": "Yes",
       "btnNo": "No"
+    },
+    "DeleteAvatarModal": {
+      "title": "Delete profile photo",
+      "question": "Do you really want delete uploaded profile photo",
```

Comment:
Question mark?

Reply:
it have to be in all questions or nowhere